packages/server/src/centrifuge-client/centrifuge.service.ts

```diff
@@ -34,7 +34,7 @@ export class CentrifugeService {
   pullForJobComplete(jobId: string, authorization: string): Promise<JobsStatusResponse> {
     return this.job.getJobStatus(authorization, jobId).then(result => {
       if (result.status === 'pending') {
-        return delay(500).then(() => this.pullForJobComplete(jobId, authorization));
+        return delay(250).then(() => this.pullForJobComplete(jobId, authorization));
```

Comment:
Any particular reason to drop the delay?

Reply:
after conversation with @mikiquantum , we agreed that 500 ms was too long and 250 ms was sufficient for the delay in checking if a job has finished on the node side